docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-sphinx>=1.7.6,<3.0
+sphinx>=1.7.6
```

Comment:
Alternative to this change (and the `schema.py` above is to pin e.g.
```
Jinja2==2.11.3
markupsafe==2.0.1
```

Reply:
With either the pinned version or this current one I do see
```
/home/rnc/Work/CEKit/GitHub/cekit/docs/descriptor/includes/image/packages.rst:1: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'descriptor/includes/image/includes/packages/common.rst'.
```

but the doc generation (and on the PR) looks ok.

Reply:
Ok so in that case going with what we have in this PR is better :-)